src/_includes/publication.html

```diff
@@ -120,6 +120,20 @@ <h5 class="g-color-black g-font-weight-600">{{ page.author | author_links}}</h5>
     </div>
 </section>
 
+{% if site.data.toggles.feature-sm-blog-in-page-banner == 'on' %}
+    {% assign posttags = page.tags %}
+    {% for posttag in posttags %}
+        {% if posttag == 'software modernisation' %}
+            <div class="container section g-pt-200 g-pb-100">
+                <div>
+                    {% include software_modernisation_in-page_banner--publications.html %}
```

Comment:
Is there any reason to keep these two `div` tags outside of `software_modernisation_in-page_banner--publications.html`?